README.md

```diff
@@ -177,7 +177,7 @@ Note that
 
 ### Change Log
 
-#### v0.0.49 (07/07/2021)
+#### v0.0.40 (07/07/2021)
```

Comment:
Should this be 50?
```suggestion
#### v0.0.50 (07/07/2021)
```

Reply:
No, the current version is 0.0.39 - I just mistyped it in the initial review.